### PR TITLE
DOCS: Add datatypes info for timestamp9 extension and add topic for isn extension

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**
Staging here:

https://docs-staging.vmware.com/en/Greenplum-Database-7/GPDOCS-326-extensions-updates/greenplum-database/GUID-ref_guide-modules-isn.html

https://docs-staging.vmware.com/en/Greenplum-Database-7/GPDOCS-326-extensions-updates/greenplum-database/GUID-ref_guide-modules-timestamp9.html

### CODE REVIEW

File: gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md [hunk 1]
```diff
@@ -38,6 +46,11 @@ testdb=# SELECT now()::timestamp9;
 (1 row)
 ```
 
+## <a id="topic_info"></a>Module Documentation 
+
+Refer to the [timestamp9 github documentation](https://github.com/fvannee/timestamp9) for detailed information about using the module.
```
Comment (higuoxing) [inline gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md:51]: Documentation in https://github.com/fvannee/timestamp9 doesn't contain these two types: `timestamp9_ltz` and `timestamp9_ntz`. We have to add documentation for these two types later.

File: gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md [hunk 2]
```diff
@@ -14,11 +14,19 @@ CREATE EXTENSION timestamp9;
...
+The Greenplum Database `timestamp9` extension supports three kinds of datatatypes: `TIMESTAMP9`, `TIMESTAMP9_LTZ` and `TIMESTAMP9_NTZ`. (The `TIMESTAMP9_LTZ` data type is an alias for `TIMESTAMP9` data type.) 
+
+The following table summarizes key information about the `timestamp9` data types:
 
-## <a id="topic_gp"></a>Additional Documentation
+|Data Type|Storage Size|Description|Max Value|Min Value|Resolution|
+|--------|-------|----|-----------|---------|---------|----------|
+|`TIMESTAMP9`|8 bytes|Like TIMESTAMP9_LTZ. Timestamp with local time zone. |2262-04-11 00:00:00.000000000 +0000 |1970-01-01 00:00:00.000000000 +0000 | 1 nanosecond |
+|`TIMESTAMP9_LTZ`|8 bytes|Timestamp with local time zone. |2262-04-11 00:00:00.000000000 +0000 |1970-01-01 00:00:00.000000000 +0000 | 1 nanosecond |
+|`TIMESTAMP9_NTZ`|8 bytes|Timestamp without time zone. |2262-04-11 00:00:00.000000000 +0000 |1970-01-01 00:00:00.000000000 +0000 | 1 nanosecond|
```
Comment (higuoxing) [inline gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md:27]: The `Max Value` and `Min Value` are not very clear in the documentation in https://github.com/fvannee/timestamp9. I'm not 100% for sure the value is correct. I'm working on a patch to that repo to help address this issue. I can help correct the value in our documentation once the issue is addressed.

File: gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 The `timestamp9` module provides an efficient, nanosecond-precision timestamp data type and related functions and operators.
 
-The Greenplum Database `timestamp9` module is based on version 1.1.0 of the `timestamp9` module used with PostgreSQL.
+The Greenplum Database `timestamp9` module is based on version 1.2.0 of the `timestamp9` module used with PostgreSQL.
```
Comment (higuoxing) [inline gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md:5]: The version of timestamp9 is bumped to 1.3.0 recently. I'm not sure if we're going to release 1.2.0 or 1.3.0 for Greenplum 6.24 release. I will help correct the version number once it's determined.

**Review by higuoxing — COMMENTED** [review]
LGTM for the timestamp9 part with some undetermined values. I will help correct the documentation once they are determined.

File: gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md (hunk 1 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md:51]: Sounds good.

File: gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md (hunk 2 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md:27]: Okay, thank you.

File: gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md (hunk 3 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md:5]: We're doing 1.2.0 for 6.24.

File: gpdb-doc/markdown/ref_guide/modules/isn.html.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# isn 
+
+The `isn` module provides support for the international product numbering standards EAN13, UPC, ISBN (books), ISMN (music), and ISSN (serials). 
+
+The Greenplum Database `isn` module is equivalent to version XXX of the `isn` module used with PostgreSQL. There are no Greenplum Database or MPP-specific considerations for the module.
```
Comment (zhrt123) [inline gpdb-doc/markdown/ref_guide/modules/isn.html.md:5]: For GPDB7: the isn module is in PostgreSQL 12, whose doc is https://www.postgresql.org/docs/12/isn.html. The version of isn is 1.2
For GPDB6: the isn module is in PostGreSQL 9.4, whose doc is https://www.postgresql.org/docs/9.4/isn.html. The version of isn is 1.0

File: gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md (hunk 2 above)
Comment (higuoxing) [inline gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md:27]: Hi, in our released extension, the value range for the timestamp is as follows.

```suggestion
|`TIMESTAMP9`|8 bytes|Like TIMESTAMP9_LTZ. Timestamp with local time zone. |2261-12-31 23:59:59.999999999 +0000 |1700-01-01 00:00:00.000000000 +0000 | 1 nanosecond |
|`TIMESTAMP9_LTZ`|8 bytes|Timestamp with local time zone. |2261-12-31 23:59:59.999999999 +0000 |1700-01-01 00:00:00.000000000 +0000 | 1 nanosecond |
|`TIMESTAMP9_NTZ`|8 bytes|Timestamp without time zone. |2261-12-31 23:59:59.999999999 |1700-01-01 00:00:00.000000000 | 1 nanosecond|
```

Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/ref_guide/modules/timestamp9.html.md:27]: Thanks, @higuoxing I will apply this update.

File: gpdb-doc/markdown/ref_guide/modules/isn.html.md (hunk 4 above)
Comment (zhrt123) [inline gpdb-doc/markdown/ref_guide/modules/isn.html.md:5]: Should we change `xxx` to `1.2`?

Comment (dyozie) [reply gpdb-doc/markdown/ref_guide/modules/isn.html.md:5]: @VMWareJennyBeth still need to commit the above change.

**Review by dyozie — CHANGES_REQUESTED** [review]

**Review by dyozie — APPROVED** [review]

**Pull request merged**